helm/idol-qms/values.yaml

```
@@ -46,12 +46,12 @@ additionalVolumes: []
 additionalVolumeMounts: []
 
 # -- Default configuration for [PromotionAgentStore]::Host
-agentStoreName: &agentStoreName "idol-agentstore-query-service"
+agentStoreName: &agentStoreName "qms-agentstore"
 # -- Default configuration for [PromotionAgentStore]::Port
 agentStoreACIPort: &agentstoreACIPort "9150"
 
 # -- Default configuration for [IDOL]::Host
-queryserviceName: &queryServiceName "idol-query-service"
+queryserviceName: &queryServiceName "idol-content"
```

```suggestion
queryserviceName: &queryServiceName "idol-query-service"
```
This should still be `idol-query-service` so that you can deploy the `idol-qms` alongside `distributed-idol` in place of enabling `single-content`